doc/classes/Parallax2D.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="Parallax2D" inherits="Node2D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A node used to create a parallax scrolling background.
+	</brief_description>
+	<description>
+		A Parallax2D is used to create a parallax effect. It can move at a different speed relative to the camera movement using [member scroll_scale]. This creates an illusion of depth in a 2D game. If manual scrolling is desired, the [Camera2D] position can be ignored with [member override_ignore_camera_scroll].
+		[b]Note:[/b] Any changes to this node's position made after it enters the scene will be ignored.
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="autoscroll" type="Vector2" setter="set_autoscroll" getter="get_autoscroll" default="Vector2(0, 0)">
+			When set, the offset will scroll automatically by the set amount of pixels per second.
+		</member>
+		<member name="limit_begin" type="Vector2" setter="set_limit_begin" getter="get_limit_begin" default="Vector2(0, 0)">
+			Top-left limits for scrolling to begin. If the camera is outside of this limit, the Parallax2D will stop scrolling. Must be lower than [member limit_end] minus the viewport size to work.
+		</member>
+		<member name="limit_end" type="Vector2" setter="set_limit_end" getter="get_limit_end" default="Vector2(0, 0)">
+			Bottom-right limits for scrolling to end. If the camera is outside of this limit, the Parallax2D will stop scrolling. Must be higher than [member limit_begin] and the viewport size combined to work.
+		</member>
+		<member name="offset" type="Vector2" setter="set_offset" getter="get_offset" default="Vector2(0, 0)">
+			The Parallax2D's offset relative to its parent.
```

Comment:
As it's the name of a built-in class, surround its name like this.
```suggestion
			The [Parallax2D]'s offset relative to its parent.
```
Should be applied to other similar mentions.

In my opinion it would be better to say "The parallax node's offset [...]" because it's more natural, but in this situation it could be better to avoid any possible ambiguity with the existing class.

Reply:
Good call out. I looked at a few other node docs as examples to pull from, but they're inconsistent, so I assumed you didn't surround it if it was the same class. Thanks for the clarification!

Reply:
> but they're inconsistent

Ooh boy, [_tell me about it_](https://github.com/godotengine/godot/pulls?q=is%3Apr+author%3Amickeon+overhaul)...